plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/FaastReader.java

```diff
@@ -93,9 +93,12 @@ private String getCategory(String url) {
         return url.substring(locator_start, locator_end);
     }
 
-    private String getTestCase(String url) {
-        int locator =
-                url.lastIndexOf(BenchmarkScore.TESTCASENAME) + BenchmarkScore.TESTCASENAME.length();
-        return url.substring(locator, locator + 5);
+    private int getTestCase(String url) {
+        // TODO: Check with more test cases
```

Comment:
I don't like new 'TODO's. Is this still relevant. I also don't want to retain commented out code if its not needed anymore.